graylog2-web-interface/src/components/common/ColorPicker.tsx

```diff
@@ -17,13 +17,21 @@
 import * as React from 'react';
 import { useCallback } from 'react';
 import PropTypes from 'prop-types';
-import { SwatchesPicker } from 'react-color';
+import { SwatchesPicker, ColorResult } from 'react-color';
```

Comment:
Should we import the type explicitly? 

```suggestion
import { SwatchesPicker } from 'react-color';
import type { ColorResult } from 'react-color';
```